### PR TITLE
feat(acceptance): adding acceptance matrix

### PR DESCRIPTION
**Pull request by enoodle**

### CODE REVIEW

File: test/acceptance/acceptance.js [hunk 1]
```diff
@@ -39,6 +71,32 @@ async function basicTest(lambdaName, input) {
...
+/**
+ * basicNoReturn
+ *
+ * @param {string} lambdaName
+ * @param {object} input
+ * @returns {undefined}
+ */
+async function basicNoReturn(lambdaName, input) {
+    const response = await invoke(lambdaName, input);
+    expect(response.StatusCode).to.equal(200);
```
Comment (galbash) [inline test/acceptance/acceptance.js:83]: Maybe assert the payload is 'null'?

File: test/acceptance/acceptance.js [hunk 2]
```diff
@@ -55,7 +113,72 @@ describe('Lambda Wrapper', () => {
...
+        });
+    });
+
+    const syncOptions = ['sync'];
+    const paramOptions = ['no', '1', '2', '3'];
+    const returnOptions = [
+        'simple_return', 'no_return',
+        'succeed', 'fail', 'done',
+        'callback', 'callback_error', 'callback_then_fail'];
+    if (RUNTIME === 'nodejs8.10') {
```
Comment (galbash) [inline test/acceptance/acceptance.js:127]: Maybe on node6.10 push promise-void or something whereyou return a promise but expect the result to be null

**Review by galbash — DISMISSED** [review]
Check out the comments

File: test/acceptance/acceptance.js (hunk 2 above)
Comment (enoodle) [reply test/acceptance/acceptance.js:127]: then it is like no returning anything which we already check..

File: test/acceptance/acceptance.js (hunk 1 above)
Comment (enoodle) [reply test/acceptance/acceptance.js:83]: :+1: will add

**Review by galbash — APPROVED** [review]

**Pull request merged**